src/App.css

```diff
@@ -21,10 +38,303 @@
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #4386bf;
+  border: none;
+  background-color: transparent;
+}
+
+.chapters {
+  display: flex;
+  flex-direction: column;
+}
+
+.curchap {
+  width: 7.875rem;
+  height: 1.125rem;
+  margin: 0.063rem 3.625rem 0 3.688rem;
+  opacity: 0.5;
+  font-family: "Montserrat", sans-serif;
+  font-size: 0.813rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: white;
+}
+
+.chap {
+  width: 5.625rem;
+  height: 1.313rem;
+  margin: 0.438rem 5.875rem 0.25rem 3.688rem;
+  font-family: "Montserrat", sans-serif;
+  font-size: 1rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.4px;
+  color: white;
+}
+
+.update {
+  width: 11.5rem;
+  height: 2.063rem;
+  margin: 0.75rem 0 0.563rem 3.688rem;
+  padding: 0.438rem 1.188rem 0.5rem 1.375rem;
+  border-radius: 3px;
+  border: none;
+  background-color: #d5a458;
+  font-family: "Montserrat", sans-serif;
+  font-size: 0.813rem;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 0.5px;
+  text-align: center;
+  color: #fff;
+}
+
+.complete {
+  width: 5rem;
+  height: 1.125rem;
+  margin: 0.438rem 4.938rem 0.75rem 0.75rem;
+  opacity: 0.5;
+  font-family: "Montserrat", sans-serif;
+  font-size: 0.875rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: white;
+}
+
+.completediv {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+}
+
+.percent {
+  width: 2.938rem;
+  height: 2.438rem;
+  margin: 0.063rem 7.063rem 0 0.688rem;
+  font-family: "Montserrat", sans-serif;
+  font-size: 2rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: white;
+}
+
+.text {
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+}
+
+.formdiv h3 {
+  width: 10.875rem;
+  height: 1.5rem;
+  margin: 1.813rem 32.75rem 1.188rem 0;
+  font-family: "Montserrat", sans-serif;
+  font-size: 1.25rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.18px;
+  color: gray;
+}
+
+.formdiv {
+  width: 75rem;
+  height: 10.625rem;
+  margin: 2.313rem 8.25rem 0;
+  justify-content: space-between;
+  padding: 2rem 9.188rem 1.625rem 1.688rem;
+  border-radius: 4px;
+  border: solid 1px var(--white);
+  background-color: #fff;
+}
+
+.input {
+  width: 20.5rem;
+  height: 2.813rem;
+  margin: 1.188rem 2.125rem 0 0;
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+  background-color: #fff;
+}
+
+.cat {
+  width: 7.813rem;
+  height: 2.813rem;
+  margin: 1.188rem 2.125rem 0;
+  padding: 0.375rem 0.938rem 0.563rem 1rem;
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+  background-color: #fff;
```

Comment:
- [ ] Great job on following the design but I think you should use the same color to match the design as much as possible 🔥🚀💯💯 since it is different in your project 👌🏻 so please make it similar 💯💯

- [ ] I think you have horeziontal scrolling in the side so please fix it since it is not found in the original design

 - Your design
 
![image](https://user-images.githubusercontent.com/58553711/187044520-843ff623-c40f-4dff-a948-37a23595c3c9.png)

- Zeplin design
![image](https://user-images.githubusercontent.com/58553711/187044535-29df1097-7278-4c09-9d5e-1bbbaefb1e32.png)